pkg/kubectl/cmd/clusterinfo.go

```diff
@@ -43,6 +43,7 @@ func NewCmdClusterInfo(f *cmdutil.Factory, out io.Writer) *cobra.Command {
 		},
 	}
 	cmdutil.AddInclude3rdPartyFlags(cmd)
+	cmd.AddCommand(NewCmdClusterInfoDump(f, out))
```

Comment:
Suggest removing `clusterinfo` (L37 & L51) since it's deprecated more than 1 year ago.